src/Core/Framework/Adapter/Twig/Extension/NodeExtension.php

```diff
@@ -16,6 +16,8 @@ class NodeExtension extends AbstractExtension
 {
     /**
      * @internal
+     *
+     * @deprecated tag:v6.7.0  - replace TemplateFinder with TemplateFinderInterface
```

Comment:
```suggestion
     * @deprecated tag:v6.8.0  - replace TemplateFinder with TemplateFinderInterface
```

it is too late for 6.7 :grin:

Reply:
Oh no :scream: Not a big problem I guess ... this can wait